module06/en.subject.tex

```diff
@@ -523,7 +523,7 @@ \section*{Examples}
 # Example 0.2:
 lr1.loss_(y, y_hat)
 # Output:
-195.34539903032385
+390.69079806064764
```

Comment:
You cannot do that because student will implement the definition of the loss from the previous module where the division by 2 is in the formula. Plus as we suggest to look to the video of the MOOC, within the video the definition also display a division by 2

Reply:
You are right ! 
Moreover, The subjects seems actually okay, both ex03 and ex04 both divide by 2 the loss.